Sources/SynKit/Benchmarks/Parser.Benchmarks/Parser.Benchmarks.csproj

```diff
@@ -8,8 +8,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="$(SolutionDir)\SynKit\Libraries\Parser\Parser.csproj" />
-    <ProjectReference Include="$(SolutionDir)\SynKit\Libraries\Lexer.Generator\Lexer.Generator.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="False" />
-    <ProjectReference Include="$(SolutionDir)\SynKit\Libraries\Parser.Generator\Parser.Generator.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="False" />
+    <PackageReference Include="BenchmarkDotNet" Version="0.13.1" />
+    <PackageReference Include="Yoakke.Synkit.Lexer"            Version="2022.6.14-8.42.51-nightly" />
+    <PackageReference Include="Yoakke.Synkit.Lexer.Generator"  Version="2022.6.14-8.42.51-nightly" />
+    <PackageReference Include="Yoakke.Synkit.Parser"           Version="2022.6.14-8.42.51-nightly" />
+    <PackageReference Include="Yoakke.Synkit.Parser.Generator" Version="2022.6.14-8.42.51-nightly" />
```

Comment:
BDN otherwise [fails to run](https://benchmarkdotnet.org/articles/guides/troubleshooting.html)